### DISCUSSION TITLE
15w laser machine has completely different settings so what is the solution please help

### DISCUSSION THREAD

**Discussion by munzerta** [arkypita/LaserGRBL#842] [Q&A]
I tried to reset the factory of the laser machine, but what happened is not what is requested. The settings of the laser machine are completely different. Please help. Thank you very much.

![IMG_0006 (1)](https://user-images.githubusercontent.com/59838836/72279949-e588aa80-363f-11ea-9e36-e3b3fd8ced85.JPG)
![IMG_0005](https://user-images.githubusercontent.com/59838836/72279950-e6214100-363f-11ea-93f4-a319aa68e7e5.JPG)

**Comment by StuartB4** [comment]
![676767](https://user-images.githubusercontent.com/16980026/72285755-b2dcb300-363b-11ea-91fc-0a39a003775e.jpg)


Start LaserGRBL and click Connect.
Then goto Grbl menu and click on Grbl Configuration. 
You should get this list of settings.
Some of them will need to be set to suit your machine.

![klkl](https://user-images.githubusercontent.com/16980026/72285602-5b3e4780-363b-11ea-88c6-0e6d3b04e810.jpg)
